nixos/modules/services/networking/zerotierone.nix

```diff
@@ -83,6 +97,16 @@ in
           ln -s ${localConfFile} ${localConfFilePath}
         '';
 
+      postStart = lib.optionalString cfg.generateSystemdNetworkdConfig ''
+        # zerotier-systemd-manager requires http://localhost:9993 to be available,
+        # which may take some short time.
+
+        sleep 5
+
+        # a failure of zerotier-systemd-manager should not take down the zerotierone service.
+        ${lib.getExe cfg.systemdManagerPackage} || true
```

Comment:
I wonder if this shouldn't be a separate service. Then it can still be shown as failed, it can restart on its own, etc... You can use the wants/needs dependencies to set the relationship.

Reply:
Usage of zerotier-systemd-manager is tightly coupled to zerotier-one, as it should be rerun every time zerotier-one is (re-)started, to check for changes in the zerotier network configuration.

While this could be realized with `PartOf=`  and `oneshot`, instead of `WantedBy`, I don't think this really merits another service, especially since that would not eliminate the need for the delay.

Adding another `service.` entry would make this feature less discoverable, and would require one depend on the other, which I am not sure how to do.